File: src/main/java/com/glitch/knowledge/service/LineService.java
package com.glitch.knowledge.service;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import com.glitch.knowledge.model.Line;
import com.glitch.knowledge.repositories.LineRepository;

@Service
public class LineService {

    @Autowired
    private LineRepository lineRepository;

    /**
     * @param line_id ID of queried line 
     * @return {@link Line} if line exists, otherwise null
     */
    public Line get(int line_id) {
        return lineRepository.findById(line_id).orElse(null);
    }

}

File: src/main/kotlin/com/glitch/knowledge/service/LineService.kt
package com.glitch.knowledge.service

import com.glitch.knowledge.model.Line
import com.glitch.knowledge.repository.LineRepository
import org.springframework.stereotype.Service

@Service
class LineService(private var lineRepository: LineRepository) {
    /**
     * @param line_id ID of queried line
     * @return [Line] if line exists, otherwise null
     */
    fun get(line_id: Int): Line? {
        return lineRepository.findById(line_id).orElse(null)
    }
}